site/src/examples/progress/HiddenLabel.tsx

```diff
@@ -0,0 +1,6 @@
+import { ReactElement } from "react";
+import { CircularProgress } from "@salt-ds/lab";
+
+export const HiddenLabel = (): ReactElement => (
```

Comment:
More of a general site example question. Do we want to show Circular and Linear progress on examples that are common

Reply:
Did we make a decision here?